lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss

```diff
@@ -31,17 +28,15 @@
           right: 0;
           opacity: 0;
           cursor: pointer;
+          width: 100%;
         }
       }
     }
   }
 
   .btn {
-    &.secondary {
-      background: $blue;
-      display: table;
-      margin: 0 auto;
-    }
+    display: table;
+    margin: 0 auto;
```

Comment:
@weerd showed me this trick to center the button. It allows you to center it without specifying a width. The button will always be centered but allow for a variable length of text.